repo.xml

```diff
@@ -8,10 +8,17 @@
     <copyright>true</copyright>
     <type>application</type>
     <target>public-repo</target>
-    <prepare>pre-install.xql</prepare>
-    <finish>post-install.xql</finish>
+    <prepare>pre-install.xq</prepare>
+    <finish>post-install.xq</finish>
     <permissions xmlns:repo="http://exist-db.org/xquery/repo" password="repo" user="repo" group="repo" mode="rw-rw-r--"/>
     <changelog>
+        <change version="2.0.0">
+            <ul xmlns="http://www.w3.org/1999/xhtml">
+                <li>IMPORTANT: You must first run the upgrade script in TODO_INSERT_URL before installing v2!</li>
```

Comment:
Note TODO - I plan to link to the module's GitHub URL - which strikes me as the most direct route to finding it for someone who hasn't yet installed v2.

Reply:
Alternatively, I could remove the script from the repository and post it as a gist.

Reply:
I've already updated the link to point to the module's GitHub URL.